Common/src/main/java/com/tabnineCommon/chat/ChatBrowser.kt

```diff
@@ -161,7 +156,28 @@ class ChatBrowser(messagesRouter: ChatMessagesRouter, private val project: Proje
         val settings = JCefAppConfig.getInstance()
         settings.cefSettings.command_line_args_disabled = false
         settings.appArgsAsList.add("--disable-web-security")
+
+        val proxyServer = getProxySettings()
+        if (proxyServer != null) {
+            settings.appArgsAsList.add("--proxy-server=$proxyServer")
+        }
+
         val cefApp = JBCefApp.getInstance()
         return JBCefBrowser(cefApp.createClient(), "")
     }
 }
+
+fun getProxySettings(): String? {
+    val httpConfigurable = HttpConfigurable.getInstance()
+
+    return if (httpConfigurable.USE_HTTP_PROXY) {
+        val auth = if (httpConfigurable.PROXY_AUTHENTICATION && httpConfigurable.proxyLogin?.isNotEmpty() == true) {
+            "${httpConfigurable.proxyLogin}:${httpConfigurable.plainProxyPassword}@"
+        } else {
+            ""
+        }
+        "${auth}${httpConfigurable.PROXY_HOST}:${httpConfigurable.PROXY_PORT}"
```

Comment:
I think this is missing the protocol(http/https)

Reply:
but there is no option in IJ settings to choose a protocol. by omitting the protocol, it should use http.

Reply:
Yes, we use http for the binary

Reply:
https://github.com/codota/tabnine-intellij/blob/1addb608344be27f228518a883420a22046f3e53/Common/src/main/java/com/tabnineCommon/binary/BinaryProcessGateway.kt#L56

Reply:
but what I did is part of the chromium protocol:
https://www.chromium.org/developers/design-documents/network-settings/#command-line-options-for-proxy-settings
we can set "http=...", but "http://..." doesn't seem to be the protocol in chromium. (http is the default so we don't have to specify it)